### DISCUSSION TITLE
[Config Support]:  Clean the storage

### DISCUSSION THREAD

**Discussion by MiAutomations** [blakeblackshear/frigate#11811] [Config Support]
### Describe the problem you are having

Hello by mistake I configure the record as true in the  frigate.yml file

I running in a out of space in the HA .... 

Now I clean all the videos from the storage inside of the \\192.168.2.xxx\media\frigate\

But I still view a lot of files in the GUI of the HA media as for example 
![image](https://github.com/blakeblackshear/frigate/assets/47602733/966d2f3a-2702-4b76-ae85-84ac23f5559c)

And also I have running a lot space ocupe without any video files  now...

![image](https://github.com/blakeblackshear/frigate/assets/47602733/b956a518-48b6-4a7a-85a5-92f66a7e207a)

Is there any way to fix this ? 

Thank you and best regards 

### Version

Current version: 0.14.0-beta2

### Frigate config file

```yaml
Not needed
```

### Relevant log output

```shell
Not needed
```

### Frigate stats

```json
Not needed
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Any other information that may be helpful

Nothing relevant

**Comment by hawkeye217** [comment]
Frigate auto-deletes recordings based on your retention settings. You can set the `retain` config option to a low value and restart Frigate to delete footage.

```
record:
  enabled: True
  retain:
    days: 1
```

Setting it at `0` will only retain footage during the time an object is being tracked.

**Reply by MiAutomations** [reply]
> Frigate auto-deletes recordings based on your retention settings. You can set the `retain` config option to a low value and restart Frigate to delete footage.
> 
> ```
> record:
>   enabled: True
>   retain:
>     days: 1
> ```
> 
> Setting it at `0` will only retain footage during the time an object is being tracked.

Hello Thank you 

I set the retain for record and also for the snapshot and I restart the addon Frigate as you mentioned, but without success

record:
  enabled: True
  retain:
    days: 0
 snapshots::
  enabled: True
  retain:
    days: 0
    
![image](https://github.com/blakeblackshear/frigate/assets/47602733/8d53c571-febd-40a8-9d18-fe3c70245be5)
    

In the GUI of the media in the HA I still have a lot of files 

![image](https://github.com/blakeblackshear/frigate/assets/47602733/c6537aa4-5be7-40a3-925d-dd7c557421e1)
